README.md

```diff
@@ -454,16 +454,36 @@ depending on whether the runtime environment is an AWS lambda.
 
 ### Workflow
 
-On PR creation:
+When a PR is created or updated:
 
-- Terraform plan generation (saved as a comment to the PR)
-- Terraform linting
-- Lambda functions linting and unit testing
+- Lint PR title
+- Lint code
+- Run unit tests
+- Run a TF plan
 
-On PR approval:
+When a PR is approved:
 
-- Terraform apply
-- Lambda functions build and deploy (only those with changes)
+- Deploy (dev)
+- Run integration tests (dev)
+
+when a PR is merged to main:
+
+- Deploy (test)
+- Run integration tests (test)
+
+When a draft release is created (manually, in Github):
+
+- GitHub draft release created with automated change notes
+
+When a draft release is published:
+
+- Deploy (prod)
+
+Workflows that permit manual dispatch:
```

Comment:
Might be nice to include a note about the hotfixes in the readme :)

Reply:
oh aye!